web_src/less/_base.less

```diff
@@ -1604,10 +1617,6 @@ table th[data-sortt-desc] {
   }
 }
 
-.dropdown .ui.label {
-  margin-left: 0 !important;
-}
-
```

Comment:
@zeripath I don't suppose you know what this was for? It came from https://github.com/go-gitea/gitea/pull/12766/files#diff-5d1373f9290f762a4751c497b0e8be279eb04caa3a7cdc2fd3f42b9f505a83be.

Reply:
ping @zeripath. I don't see anything obvious wrong on the graph dropdown after removing that rule, so I guess we can go ahead with this removal.

Reply:
cool